en/getting-started.md

```diff
@@ -1,10 +1,106 @@
+
+The key usually follows a global pattern and can be easily created automatically by clicking on the "generate" button next to the field.
+
 {% hint style="info" %}
-Coming soon!
+The default key pattern is `[auth][year]`, which means that Author information is followed by the year of the publication, resulting in the example in `Turing1950`. However, the key pattern is customizable to your needs. See [Configuration](https://docs.jabref.org/setup) > ["Customize the citation key generator"](https://docs.jabref.org/setup/bibtexkeypatterns) for more details.
 {% endhint %}
 
-Desktop interface \(basic description of the user interface elements and what they do\)
+After entering some information, you can see on the right side of the entry editor a preview of the bibliographic data:
+
+![Added information for new entry](.gitbook/assets/getting-started-filled-entry-editor.png)
 
-Quick start \(basic userflow: create new library -&gt; add entry, link file, do web search -&gt; save\)
+## Enhancing the information
 
+After creating the basic information the addition of all other bibliographical details is often cumbersome and error-prone. To ease this task, JabRef allows for an automatic completion of the bibliographic information by looking up the data in public databases.
+To use this feature just click on the "Update with bibliographic information from the web" button in the editor:
+
+![Update information from web](.gitbook/assets/getting-started-entry-editor-update-from-web.png)
+
+{% hint style="info" %}
+The found information is most accurate if an identifier like a "DOI" or "ISBN" is maintained. If you already know such an unique identifier, this can also be already the starting point to create a new entry without manual entering any information by using the "create from ID" feature in the Create entry dialog. For more information see: [Collect](https://docs.jabref.org/collect) > ["Add entry using an ID"](https://docs.jabref.org/collect/add-entry-using-an-id)
+{% endhint %}
+
+If additional information is found you will be asked in a dialog which information should be taken over:
+
+![Merging the existing and the web information](.gitbook/assets/getting-started-merge-entries.png)
+
+## Adding a full text document
+
+Usually you also want to attach a reference to the full-text of a reference. For this, you can use the "file" field in the "General" tab.
+Here you can either attach a file manually, search for an already existing local file matching the bibtexkey pattern, or trying to automatically download a matching full text from the web.
+
+{% hint style="info" %}
+In order to use the automated feature, it is necessary to set-up a file directory first. To do so, please go to "Options" > "Preferences", go to "File" section and select there an existing folder as the "Main file directory":
```

Comment:
```suggestion
In order to use the automated feature, it is necessary to set-up a file directory.  The file directory can either be global, local to a user, or specific for a library. See the help on [Override file directories](https://docs.jabref.org/setup/databaseproperties#override-default-file-directories) for details. 
For beginners, a main file directory is the most simple setup. It can be set in "Options" -> "Preferences" -> "File"  
```

Reply:
The default behavior is that JabRef looks for files relative to the bib file, so I would describe this in the getting started section, an then point them to the "Override file directories" section if they want to change that.

If we need to point users to the preferences in "getting started", then we should change the default preferences in my opinion.

Reply:
Unfortunately, this is not the case - I actually reset the settings before creating the screenshots - and without and change in the settings the following dialog appears:
![grafik](https://user-images.githubusercontent.com/676652/92331417-b57e9380-f076-11ea-8032-1c0d135decfa.png)

Default Prefs setting: 
![Grabbed_20200906-192612](https://user-images.githubusercontent.com/676652/92331434-e232ab00-f076-11ea-9b7e-f0ff5f5e0018.png)

Reply:
Yep. That's a bug which was somehow introduced in 5.1. No idea how it came.